templates/zerver/integrations/errbot.md

```diff
@@ -1,18 +1,21 @@
+# Zulip Errbot integration
+
 Run your favorite chatbot in Zulip!
 
-1. [Install errbot](https://errbot.readthedocs.io/en/latest/user_guide/setup.html)
-   and follow to instructions to set up a `config.py`.
+{start_tabs}
 
-1. Check out our [Errbot integration package for Zulip](https://github.com/zulip/errbot-backend-zulip).
-   Clone this repository somewhere convenient.
+1. [Install errbot][install-errbot], and follow the instructions to set up a
+   `config.py`.
 
-1. Install the requirements listed in `errbot-backend-zulip/requirements.txt`.
+1. Clone our [Errbot integration package for Zulip][errbot-package]
+   repository somewhere convenient, and install the requirements listed in
+   `errbot-backend-zulip/requirements.txt`.
 
 1. {!create-a-generic-bot.md!}
+   Download its `zuliprc` config file.
```

Comment:
We should probably have a Related documentation section for this one with the ErrBot doc link, the repository link and the API doc on configuring python bindings since we mention the `zuliprc` config file here.

Reply:
The current formatting is kind of odd. The download instruction can be a separate step.